content/javascript/general-authoring/index.md

```diff
@@ -408,6 +408,25 @@ function yup(name, options, args) {
 }
 ```
 
+### Arrow Functions
+
+How to use arrow functions
+- Don't wrap a single argument with parenthesis.
+- When returning a function from an arrow function, create a block rather than returning on one line (helps readability).
+- If support for older browsers is required, use Babel to compile arrow functions to older 
```

Comment:
this line ending at the word "older" feels a little weird to me, maybe add "syntax" or something to clarify what we're compiling to?